### PR TITLE
fix Capture Helper warning

### LINKED ISSUE #8: Implicit conversion from enumeration type
We are getting a build warning due to incompatible enums on the latest build 

`SktCaptureHelper.m:1234:21: Implicit conversion from enumeration type 'enum SKTCaptureDeviceType' to different enumeration type 'SKTCapturePropertyType' (aka 'enum SKTCapturePropertyType')
`

Should the following line... 

`property.Type = SKTCaptureDeviceTypeNone;`

Actually be...

`property.Type = SKTCapturePropertyTypeNone;`

Screenshot 2021-07-16 at 13 04 53

### PR DESCRIPTION
**Pull request by EricGlaenzer**
update the README to include the manual process to install the SDK without using cocoapods
closes #8 , #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,20 +20,29 @@ test and integrate the SDK even before getting a Socket Mobile scanner, using th
...
-* [Device Notifications (Battery Level)](#device-notifications-battery-level)
-* [SoftScan](#softscan)
-* [Closing Capture](#closing-capture)
+## Table of Contents
+
+- [Capture SDK Version 1.3.60](#capture-sdk-version-1360)
+  - [Table of Contents](#table-of-contents)
+  - [Quick install notes](#quick-install-notes)
+  - [Capture usage](#capture-usage)
+    - [1 getting a CaptureHelper instance](#1-getting-a-capturehelper-instance)
```
Comment (cyrille-socket) [inline README.md]:
```suggestion
    - [1 Getting a CaptureHelper instance](#1-getting-a-capturehelper-instance)
```

File: README.md [hunk 2]
```diff
@@ -366,3 +388,49 @@ If the motive of closing Capture is to not receive any decoded data from the
...
+   - CoreBluetooth.framework
+   - AVFoundation.framework
+   - AudioToolbox.framework
+   - ExternalAccessory.framework
+
+6. In Target->Build Settings, add to the `Header Search Paths` the headers path of the header files of the framework:
+
+   `relative/path/to/your/app/CaptureAPI/SKTCapture.xcframework/ios-arm64_i386_x86_64-simulator/Headers`
+
+7. Add app the following key-value into the Info.plist of your project:
```
Comment (cyrille-socket) [inline README.md]:
```suggestion
7. Add the following key-value into the Info.plist of your project:
```

File: README.md [hunk 3]
```diff
@@ -366,3 +388,49 @@ If the motive of closing Capture is to not receive any decoded data from the
...
+   Add an item into the `Supported external accessory protocols` key: `com.socketmobile.chs`
+
+   ![Info.plist](Docs/Infoplist.jpg)
```
Comment (cyrille-socket) [inline README.md:432]: Is the image added into the Docs folder?

**Review by cyrille-socket — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (EricGlaenzer) [reply README.md:432]: I added the missing file in the Docs folder

**Review by EricGlaenzer — COMMENTED** [review]
accepted all the suggestions and added the missing jpg file in the Docs folder

**Review by cyrille-socket — APPROVED** [review]

**Pull request merged**